src/Commands/Visit.php

```diff
@@ -84,7 +87,7 @@ protected function open($url, OutputInterface $output) {
       $browser = 'start';
     }
 
-    if ($browser) {
+    if ($browser && !isset($GLOBALS['__PHPUNIT_BOOTSTRAP'])) {
```

Comment:
This is kind of ugly. Do we have a better way to do this?

Reply:
you could put a constant in phpunit.xml and the `!defined()`

Reply:
That's almost the same thing :D. I tested it locally it works. We can improve it later.